### ISSUE TITLE
No fixtures detected when a cwd path includes dotfile.

### ISSUE THREAD

**Issue by maruware** [react-cosmos/react-cosmos#610]
### What's up?

No fixtures detected when a cwd path includes dotfile.
e.g. /Users/maruware/.ghq/github.com/maruware/project-x

### Mkay, tell me more...

function ```findFixtureFiles``` consists
* glob with absolute
* filter with micromatch

```js
export async function findFixtureFiles(
  args: Args
): Promise<Array<FixtureFile>> {
  const { fileMatch = defaultFileMatch, cwd = process.cwd() } = args || {};

  const allPaths = await globAsync('**/*', {
    cwd,
    absolute: true,
    ignore: '**/node_modules/**'
  });
  const fixturePaths = micromatch(allPaths, fileMatch);
//...
```

The problem may be solved with
* glob with no absolute
* micromatch with [dot option](https://www.npmjs.com/package/micromatch#optionsdot)

**Comment by skidding** [comment]
@maruware thanks for bringing this up, I wasn't aware of it!

> micromatch with dot option

Sounds line an easy fix.

>  glob with no absolute

Not sure I understand how this would work. Do you have a favorite solution out of the two?

**Comment by xavcz** [comment]
Solved in #618, will be released under `react-cosmos@4` asap 👍
